docs/_managedk8s/FAQ/automatic_dns/index.en.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Can I use external-dns and openstack designate for automatic dns?
+lang: "en"
+permalink: /managedk8s/faq/automatic_dns
+nav_order: 4000
```

Comment:
the nav order needs to be set :-) to a different value.

The global topic gehts the X000 number and the sub topic normal XY00.

so at least 4100 would be needed here